src/styles/layout.scss

```diff
@@ -20,7 +20,3 @@
     padding-top: ($spacer * 6);
   }
 }
-
-.main {
```

Comment:
@duranmla what's the reason behind this change? I changed to move this responsibility to the Layout (which is used in all pages)

Reply:
Check the PR description. I made special emphasis in order for you to know why. 173d1ee also the commit message have information about it.

Reply:
Indeed, it would be great if we can have the responsibility of that space in a "layout" level but I was aiming to fix this as easy as I can and if this start to be a "problem to maintain" or you have a better idea that also meets the expectations without break the hero we can do it.